lib/internal/bootstrap/pre_execution.js

```diff
@@ -332,6 +335,61 @@ function initializeDeprecations() {
     enumerable: false,
     configurable: true
   });
+
+  let warnedForProto = false;
+  const ObjectPrototype = ObjectGetPrototypeOf({});
+  function warnOnProto() {
+    if (!warnedForProto) {
+      warnedForProto = true;
+      process.emitWarning('usage of Object.prototype.__* properties are a' +
+        ' common security concern, use static methods on Object instead');
+    }
```

Comment:
Perhaps we should make these proper runtime deprecations?
/cc @addaleax

Reply:
I'm not opposed, is it just adding to the docs a new number to get the DEP###?

Reply:
added to DEP doc, idk if there is a process to do instead

Reply:
Look for other DEP codes in the code and you'll see how those are emitted :-)

Reply:
Example: https://github.com/nodejs/node/blob/e46c680bf2b211bbd52cf959ca17ee98c7f657f5/lib/sys.js#L28-L29

As soon as this is changed to emit a deprecation warning, I'll launch a CITGM run with `--throw-deprecation`

Reply:
this seems difficult to wrangle without giving a unique DEP id to each accessor given the other feedback of giving more actionable feedback in https://github.com/nodejs/node/pull/39824#discussion_r693357853